gematria/datasets/convert_bhive_to_llvm_exegesis_input.cc

```diff
@@ -60,26 +60,35 @@ int main(int argc, char* argv[]) {
     return 1;
   }
 
+  std::string initialRegValStr;
+  {
+    // initialize initialRegValStr with kInitialRegValue in hex
+    std::stringstream ss;
+    ss << std::hex << kInitialRegVal << std::dec;
+    ss >> initialRegValStr;
+  }
   std::string register_defs_lines;
   const std::unique_ptr<gematria::LlvmArchitectureSupport> llvm_support =
       gematria::LlvmArchitectureSupport::X86_64();
   const llvm::MCRegisterInfo& MRI = llvm_support->mc_register_info();
 
   // Iterate through all general purpose registers and vector registers
   // and add them to the register definitions.
+  // TODO(9Tempest): Change GR64_NOREXRegClassID to GR64_NOREX2RegClassID to
+  // include r9-r15
```

Comment:
I don't think it's really correct to say that we're including r9-r15 by switching the register class. I'd instead writing something like:
```c++
// TODO(9Temptest): Change GR64_NOREXRegClassID to GR64_NOREX2RegClassID when the LLVM version
// is bumped to avoid including the new APX GPRs (r16-r31) that have recently been added to LLVM.
```